java/README.md

```diff
@@ -0,0 +1,149 @@
+# Installation
+
+To install the Java code, do `mvn install` from the `java` directory in this repository using Apache Maven.
+Repeat this if you pull an updated version of the code. You can run the tests,
+which should output the examples, by doing `mvn test`.
+
+# Using the AIF Library
+
+To use the library in Java, include the library generated by the
+installation above into your build script or tool.  For gradle, for
+example, include the following in the dependencies in your `build.gradle` file:
+
+`dependencies {
+    compile 'com.ncc:aida-interchange:1.1.0-SNAPSHOT'
+}`
+
+In your code, import classes from the `com.ncc.aif` package.
+Then, create a model, add entities, relations, and events to the
+model, and then write the model out.
+
+The file `src/test/java/com/ncc/aif/ExamplesAndValidationTests.java`
```

Comment:
I didn't include `java` because this README is within the current directory of `src`. I debated including it, but ultimately decided on this. I am open to doing either.

Reply:
Understood...I don't feel strongly either way.  I'm fine leaving these paths as you have it.